AWSComprehendTests/AWSComprehendTests.swift

```diff
@@ -50,8 +54,12 @@ class AWSComprehendTests: XCTestCase {
         detectDominantLanguageRequest!.text = "El sol brilla en Seattle";
         
         comprehendClient.detectDominantLanguage(detectDominantLanguageRequest!).continueWith { (task) -> Any? in
+            if let error = task.error {
+                XCTAssertNil(error)
```

Comment:
Since this statement is for better understanding of error in the test report, can we do something like this 
```
XCTFail("Should not produce error - \(error)")
```

Reply:
I'll see if stringifying the error produces the same level of error detail as simply dumping it in the `XCTAssertNil`. If not, then I can change the message to `XCTAssertNil(error, "Should not produce error")`